app/admin/books.rb

```diff
@@ -3,21 +3,47 @@
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   #
   # has_many :users, through: :ordered_books
-  permit_params :title, :description, :recommended_church, :price
+  permit_params :title, :description, :recommended_church, :price, :picture
```

Comment:
@resivalex @isabloCh what bothers me is that I don't see any complaints from codeclimate for this code. I executed rubocop for this file:

```
$ rubocop app/admin/books.rb
Inspecting 1 file
C

Offenses:

app/admin/books.rb:1:1: C: Metrics/BlockLength: Block has too many lines. [36/25]
ActiveAdmin.register Book do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/admin/books.rb:9:14: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
    f.inputs "Books" do
             ^^^^^^^
app/admin/books.rb:18:1: C: Layout/TrailingWhitespace: Trailing whitespace detected.
app/admin/books.rb:22:7: C: Layout/LeadingCommentSpace: Missing space after #.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/admin/books.rb:22:8: C: Style/CommentAnnotation: Annotation keywords like ToDo should be all upper case, followed by a colon, and a space, then a note describing the problem.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
       ^^^^^
app/admin/books.rb:22:13: C: Style/AsciiComments: Use only ascii symbols in comments.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
            ^
app/admin/books.rb:38:31: C: Layout/TrailingWhitespace: Trailing whitespace detected.
    column :recommended_church 
                              ^
app/admin/books.rb:48:1: C: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body end.

1 file inspected, 8 offenses detected
```

and you can see it shoes issues. @resivalex do you think rubocop is not setup correctly for codeclimate? 

In out app it looks like that:

```
version: 2
plugins:
  rubocop:
    enabled: true
    channel: rubocop-0-52
    config:
      file: .rubocop.yml
```

Reply:
I think the rubocop setup is ok. Config files `.rubocop.yml` and `.rubocop_todo.yml` allow breaking these rules

Reply:
@resivalex ah I see!